Sources/JSONMapper/Models/JSONAPIObject.swift

```diff
@@ -0,0 +1,67 @@
+//  JSONAPIObject.swift
+//
+//  Created by Pirush Prechathavanich on 4/4/18.
+//  Copyright © 2018 Nimble. All rights reserved.
+//
+
+import Foundation
+
+public enum JSONAPIResponseType {
+
+    case data(DataType<Resource>)
+    case errors([JSONAPIError])
+    case meta(JSON)
+}
+
+public struct JSONAPIObject: Decodable {
+    //todo:- data, errors, meta - (either data or errors)
+    //       jsonapi, links, included (optional)
```

Comment:
I could be wrong, but it seems we already implemented this todo. Shall we remove it as well?